lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -278,9 +278,9 @@ def _setupInternal(self):
             return
         if self._clean_restart:
             LOG.info("Qos Setup: clean start")
+            self.impl.setup()
```

Comment:
setup and then right after destroy?

Reply:
This is to cleanup the tc objects. I could post separate PR to fix the init sequence.

Reply:
sure as long as it works